src/index.js

```diff
@@ -430,9 +431,13 @@ const compile = (schema, root, opts, scope, basePathRoot) => {
 
       if (node.pattern) {
         enforceRegex(node.pattern)
-        const p = patterns(node.pattern)
-        errorIf('!%s.test(%s)', [p, name], 'pattern mismatch')
+        if (node.pattern !== '^[\\s\\S]*$' && node.pattern !== '^[\\S\\s]*$') {
```

Comment:
What about `/^.*$/`? That seems like a simpler no-op regex.

Reply:
Ah: https://stackoverflow.com/questions/6711971/regular-expressions-match-anything